app/graphql/mutations/coupons/destroy.rb

```diff
@@ -13,7 +13,8 @@ class Destroy < BaseMutation
       field :id, ID, null: true
 
       def resolve(id:)
-        result = ::Coupons::DestroyService.new(context[:current_user]).destroy(id)
+        coupon = context[:current_user].coupons.find_by(id:)
```

Comment:
why do we chose to remove it from the `DestroyService`

Reply:
to not have `destroy_from_api` and `destroy` methods anymore ;)